client/client-multi/client-c/src/kaa/platform-impl/stm32/leafMapleMini/esp8266/esp8266_kaa_client.c

```diff
@@ -70,7 +70,9 @@ static kaa_service_t OPERATIONS_SERVICES[] = { KAA_SERVICE_PROFILE
                                              , KAA_SERVICE_LOGGING};
 static const int OPERATIONS_SERVICES_COUNT = sizeof(OPERATIONS_SERVICES) / sizeof(kaa_service_t);
 
-
+/* Logging constraints */
+#define MAX_LOG_COUNT           SIZE_MAX
+#define MAX_LOG_BUCKET_SIZE     (2 * 1024)
```

Comment:
Please, check whether KAA_MAX_BUCKET_SIZE doesn't exceed the size of a OUT transport buffer.
See defaults.h